docs/file-structure.md

```diff
@@ -36,9 +36,9 @@ Another common thing that makes components bloated is defining multiple function
 2. If anyone decides that they can reuse that code it's a quick and easy move to a shared folder.
 3. It can be confusing when you are looking for a component that is exported from a file that does not share the same name. It sounds trivial but it removes just a little bit of friction.
 
-### Use your judgement when splitting out components
+### Use your judgment when splitting out components
```

Comment:
Sounds reasonable to me - while both spellings are technically valid, `judgement` is the old British English and now rarely used spelling.

Reply:
Unless you're an Aussie! 😁